### PR TITLE
Updated Readme and separated Zeppelin setup to another page

### PR DESCRIPTION
**Pull request by ishuverma**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+This demo uses the CDK platform: learn how to install CDK https://github.com/redhatdemocentral/cdk-install-demo[here]
+
+*Please note: This repository is currently under development and the feature set/components are subject to change without notice*
+
+
+== Core Components
+*https://access.redhat.com/documentation/en/red-hat-xpaas/version-0/red-hat-xpaas-fuse-integration-services-image/[Fuse Integration Services]*
+Red Hat JBoss Fuse Integration Services provides a set of tools and containerized xPaaS images that enable development, deployment, and management of integration microservices within OpenShift.
+
+* [AMQ and MQTT](https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/)
```
Comment (sabre1041) [inline README.adoc]: These links are markdown syntax instead of asciidoc

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
+= IoT OpenShift Demo
+
+:Author:    Andrew Block, Ishu Verma
+:Email:     ablock@redhat.com, iverma@redhat.com
+:Date:      10/10/2016
+
+== Overview
+Demonstration of Intelligent IoT Gateway on the Red Hat OpenShift Container Platform. The Gateway enables IoT data transformation, routing and real time decision-making at the edge. OpenShift provides secure enterprise-grade container platform to accelerate IoT application delivery. Each component of the demo runs in its own container, thereby, allowing security, scalability and ease of deployment.
+
+image:images/ose-iot.png[width="500", height="400", align="center"]
```
Comment (sabre1041) [inline README.adoc:10]: Awesome photo. Can we include A-MQ as the recipient of the messages and fuse reading from the mqtt broker in A-MQ

File: iot-ose(2).json [hunk 3]
```diff
@@ -0,0 +1 @@
...","61.50623499837298","2016-10-01 17:38:42"],["Vibration","0.0","2016-10-01 17:38:42"],["Temperature","62.80717623303909","2016-10-01 17:38:47"],["Vibration","0.29097079333317555","2016-10-01 17:38:47"],["Temperature","64.34283672189633","2016-10-01 17:38:52"],["Vibration","1.7429542120099748","2016-10-01 17:38:52"],["Temperature","66.24462713447625","2016-10-01 17:38:57"],["Vibration","1.8557530944468408","2016-10-01 17:38:57"],["Temperature","66.80794296792833","2016-10-01 17:39:02"],["Vibration","3.2055096270271974","2016-10-01 17:39:02"],["Temperature","68.78763215976333","2016-10-01 17:39:07"],["Vibration","3.6388992551578387","2016-10-01 17:39:07"],["Temperature","70.68267522446935","2016-10-01 17:39:12"],["Vibration","5.036571517017691","2016-10-01 17:39:12"],["Temperature","72.67343404915837","2016-10-01 17:39:17"],["Vibration","5.558358923970779","2016-10-01 17:39:17"],["Temperature","74.56998717903441","2016-10-01 17:39:22"],["Vibration","7.259834223242555","2016-10-01 17:39:22"],["Temperature","50.0","2016-10-01 17:39:27"],["Vibration","8.865494110818236","2016-10-01 17:39:27"],["Temperature","51.77669183235463","2016-10-01 17:39:32"],["Vibration","10.493133069852147","2
```
Comment (sabre1041) [inline iot-ose(2).json:1]: I updated/reduced the content in my latest [PR](https://github.com/sabre1041/iot-ose/pull/10)

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Added a few comments. The latest PR and enhancements to Zeppelin will require modifications to some of the instructions. 

Great start! Let's discuss how to integrate both of these changes

File: README.adoc (hunk 2 above)
Comment (ishuverma) [reply README.adoc:10]: yes, how does this one look
iot-ose-picture

**Pull request closed without merging**